src/Stratis.Bitcoin.Features.Consensus/ProvenBlockHeaders/ProvenBlockHeaderRepository.cs

```diff
@@ -147,7 +147,7 @@ public Task PutAsync(SortedDictionary<int, ProvenBlockHeader> headers, HashHeigh
 
             if ((this.provenBlockHeaderTip != null) && (newTip.Hash == this.provenBlockHeaderTip.GetHash()))
             {
-                this.logger.LogTrace("(-)[BLOCKHASH_MISMATCH]");
+                this.logger.LogTrace("(-)[BLOCKHASH_MISMATCH]{0}:{1},{2}:{3})", nameof(newTip), newTip, nameof(this.provenBlockHeaderTip), this.provenBlockHeaderTip);
```

Comment:
Please separate this into 2 logs. One with the tips and another as an exit logs. 
Also add `''` for the tips